src/js/pensions/config/form.js

```diff
@@ -110,7 +110,8 @@ const {
 const nonRequiredFullName = createNonRequiredFullName(fullName);
 
 function isUnder65(formData) {
-  return moment().startOf('day').subtract(65, 'years').isBefore(formData.veteranDateOfBirth);
+  return moment().startOf('day').subtract(65, 'years')
+    .isBefore(formData.veteranDateOfBirth);
```

Comment:
There's a rule about breaking chained calls into multiple lines. It seems weird, but it also only seemed to cause problems in two places, so I left it on.